src/results/proforma.jl

```diff
@@ -117,22 +106,92 @@ function proforma_results(p::REoptInputs, d::Dict)
         if p.s.generator.existing_kw > 0
             fixed_and_var_om_bau = d["Generator"]["year_one_fixed_om_cost_before_tax_bau"] + 
                                    d["Generator"]["year_one_variable_om_cost_before_tax_bau"]
+            fixed_and_var_om -= fixed_and_var_om_bau
```

Comment:
@Bill-Becker I think we only want to subtract out the _bau costs if third_party is true. How it is now, for a *direct purchase* scenario the "optimal" `om_series` for the site would only account for new O&M costs, while we would want them to be all O&M costs. So I think we just need a third party if statement here, like there is in `update_metrics` for PV and Wind

Reply:
(Also I'm realizing a previous comment suggested removing the 3rd party if statement, but I think that was only applicable before this separation of new vs existing o&m costs.

Reply:
Oh right, done!